.gitignore

```diff
@@ -2,4 +2,6 @@ build/
 logs/
 nbproject/
 vendor/
+.idea/
 composer.lock
+composer.phar
```

Comment:
In fact I recommend installing composer globally to avoid this problem altogether.